app/javascript/components/schema/SignUpSchema.js

```diff
@@ -70,7 +70,13 @@ const SignUpSchema = {
   address: Joi.string().allow(''),
   city: Joi.string().allow(''),
   state: Joi.string().allow(''),
-  country: Joi.string().allow(''),
+  country: Joi.string().required().options({
```

Comment:
Nice! This is the first time I seen validations on the frontend for anything other than a MERN stack, how does Joi work in the application?

Reply:
Here Joi is used to validating the form input fields and we have added country as required as well.